pilot/pkg/networking/core/listener_waypoint.go

```diff
@@ -85,7 +85,7 @@ func (lb *ListenerBuilder) buildWaypointInbound() []*listener.Listener {
 	var wps *waypointServices
 	var forwarder *listener.Listener
 	if features.EnableAmbientMultiNetwork && isEastWestGateway(lb.node) {
-		wps = nil // TODO: implement service export functionality
+		_, wps = findWaypointResources(lb.node, lb.push)
```

Comment:
It's now occurring to me that we could probably add e2es for the `TestDirect` codepath for accessing a service through the e/w gateway now. Can you take a stab at building this out? It would be under the ambientmultinetwork conditional [here](https://github.com/istio/istio/blob/0fb4860ebce90d2c5380125f182801d60b6a9d37/tests/integration/ambient/baseline_test.go#L3149). We'd basically want to hit each clusters e/w gateway